pkg/service/replay/replay.go

```diff
@@ -564,7 +572,13 @@ func (r *Replayer) RunTestSet(ctx context.Context, testSetID string, testRunID s
 	var loopErr error
 	utils.TemplatizedValues = conf.Template
 
-	for _, testCase := range testCases {
+	for idx, testCase := range testCases {
+
+		// check if its the last test case running
+		if idx == len(testCases)-1 && r.isLastTestSet {
```

Comment:
Please check if you ignore test cases or select particular test case is this condition len(testCases)-1 is going to create any problem.